### PR TITLE
fix: define origin in setupworkplace.ts

### PR DESCRIPTION
**Pull request by LavenderFoxxo**
Self explanatory.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -6,5 +6,6 @@
 	"editor.formatOnPaste": false,
 	"editor.formatOnSave": false,
 	"editor.formatOnType": false,
-	"WillLuke.nextjs.hasPrompted": true
+	"WillLuke.nextjs.hasPrompted": true,
+	"typescript.tsdk": "node_modules\\typescript\\lib"
```
Comment (marinofranz) [inline .vscode/settings.json:10]: Remove property, could cause issues for others

File: pages/api/setupworkspace.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ export async function handler(
 	res: NextApiResponse<Data>
 ) {
 	if (req.method !== 'POST') return res.status(405).json({ success: false, error: 'Method not allowed' })
-	let userid = await getRobloxUserId(req.body.username).catch(e => null) as number | undefined;
```
Comment (marinofranz) [inline pages/api/setupworkspace.ts:31]: Change back to `null`, we don't want to log the error, that way, it just spams the console.

**Review by marinofranz — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (marinofranz) [reply .vscode/settings.json:10]: The .tsdk property if we're talking specifics

File: pages/api/setupworkspace.ts (hunk 2 above)
Comment (LavenderFoxxo) [reply pages/api/setupworkspace.ts:31]: Whoops. There we go.

**Review by marinofranz — APPROVED** [review]
lgtm

**Pull request merged**